src/site/content/en/react/react.11tydata.js

```diff
@@ -7,46 +7,27 @@ module.exports = {
     slug: 'react',
     cover: '/images/collections/react.svg',
     title: 'React',
-    updated: 'April 28, 2019',
+    updated: 'October 11, 2019',
     description: `Build performant and progressive React applications.`,
     overview: `In this section you'll learn how to use built-in APIs and
     third-party libraries to improve the performance of your
     React applications.`,
     topics: [
       {
-        title: 'Introduction',
+        title: 'Next.js',
         pathItems: [
-          'get-started-optimize-react',
+          'performance-as-a-default-with-nextjs',
```

Comment:
To be honest, I was still kind of unsure how to categorize things and was thinking of an entirely separate Next.js collection. But this looks even nicer in my opinion - I love the idea of separating CRA and Next.js optimization guides and keeping it under a single React collection. 

There's a couple of things we probably should consider doing:

- A top-level mini guide that explains that Next.js and CRA is split up here and why (and what readers should except from either)
- Moving some general guidance not specific to CRA or Next.js (like `virtualize-long-lists-react-window`) to a general section

I can do either of these later when we have more guides landed, but just throwing ideas out there!

Reply:
SGTM. I created #1674 to follow up on this.